demo/src/app/app.component.html

```diff
@@ -22,7 +22,8 @@
   </div>
 
   <div class="demo-buttons">
-    <button class="btn-demo btn-demo__clear" (click)="dateExample=null;timeExample=null;dateTimeExample=null;" >Clear All</button>
+    <button class="btn-demo btn-demo__clear" (click)="dateExample=null;timeExample=null;dateTimeExample=null;">Clear
+      All</button>
```

Comment:
Hmm :/

Reply:
There are a few instances where I'm not sure I like how the formatter formatted the `html` files.

Most `js`/`ts`/`json` files look fine, from what I saw. But `html` had a ton of issues.

Reply:
Fixed